modal/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "9567cd54-7fac-46d8-84ce-0b00cef6937e",
+  "app_id": "modal",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Troubleshooting",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect logs and metrics for your Modal applications",
+    "title": "Modal",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Modal dashboard",
+        "image_url": "images/modal-product-hero.jpg"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Cloud",
+      "Category::Metrics",
+      "Category::Log Collection",
+      "Submitted Data Type::Metrics",
+      "Submitted Data Type::Logs",
+      "Offering::Integration"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "Modal": "assets/dashboards/modal_overview.json"
+    }, 
+    "integration": {
+      "source_type_name": "Modal",
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "metadata_path": "metadata.csv",
+        "check": [],
```

Comment:
Please add a check metric. The purpose of the check metrics is that if this metric is received, Datadog will consider the Modal integration to be properly working.

Reply:
I'm not sure what is appropriate as a check metric for this log-only integration. We're not emitting metrics to datadog yet, simply logs through the log-http endpoint.

Reply:
Ah i see. If this integration is not submitting metrics, you can remove this `metrics` object

Reply:
@JoshPatel13 can you confirm ?

Reply:
In this case remove the key and value (``` "metrics": {
        "metadata_path": "metadata.csv",
        "check": [],
        "prefix": "modal."
      },```)

Reply:
Done.